### ISSUE TITLE
Fix epydoc pydoctor errors in twisted.protocols

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8553]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)| @adiroiban reported|
|-|-|
|Trac ID|trac#8553|
|Type|enhancement|
|Created|2016-07-03 15:11:32Z|
|Branch|https://github.com/twisted/twisted/tree/8553-pydoctor-t-protocols|

<details><summary>Searchable metadata</summary>

```
trac-id__8553 8553
type__enhancement enhancement
reporter__adiroiban adiroiban
priority__normal normal
milestone__None None
branch__8553_pydoctor_t_protocols 8553-pydoctor-t-protocols
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1467558692249567 1467558692249567
changetime__1467727005263074 1467727005263074
version__None None
owner__adiroiban adiroiban

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_1"></a>|@adiroiban removed owner|
|-|-|

Check that new documentation make sense. I am not sure if the AMP docstring is ok

Check that apidocs build is clean

```
./bin/admin/build-apidocs . apidocs/ | grep twisted.protocols
```

the zope interface errors are fixed in a separate ticket... which is also in the review queue.

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_2"></a>|@rodrigc set owner to @adiroiban|
|-|-|

Fix the spelling of length here:

```
@var MAX_VALUE_LENGTH: The maximum lenght of a message.
```

then OK to merge

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_3"></a>|@adiroiban set status to `closed`|
|-|-|

In changeset a13215aed7a22ecf0fb4e4e012842021d9737c05
```
#!CommitTicketReference repository="" revision="a13215aed7a22ecf0fb4e4e012842021d9737c05"
Merge 8553-pydoctor-t-protocols : Fix pydoctor errors in twisted.protocols.

Author: adiroiban
Reviewer: rodrigc
Fixes: #8553
```
